### PR TITLE
[Misc] Fix debug log empty messages

### PR DESCRIPTION
**Pull request by XingZYu**
As mentioned in [discussion#2548](https://github.com/WasmEdge/WasmEdge/discussions/2548)

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: lib/vm/vm.cpp [hunk 1]
```diff
@@ -24,8 +24,8 @@ createPluginModule(std::string_view PName, std::string_view MName) {
       return Module->create();
     }
   }
-  spdlog::debug("Plugin: "sv, PName, " , module name: "sv, MName,
-                "not found. Mock instead."sv);
+  spdlog::debug("Plugin: {} , module name: {} not found. Mock instead.", PName,
```
Comment (hydai) [inline lib/vm/vm.cpp:27]: The end should be `"sv`.

**Review by hydai — CHANGES_REQUESTED** [review]

**Comment by hydai** [comment]
Please fix the lint issue.

**Review by q82419 — APPROVED** [review]

**Review by q82419 — CHANGES_REQUESTED** [review]
Please apply the clang-format.

**Review by hydai — APPROVED** [review]

**Review by q82419 — APPROVED** [review]

**Pull request merged**